network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube_regrets_page.html

```diff
@@ -0,0 +1,42 @@
+{% extends "pages/base.html" %}
```

Comment:
Can we change the indentation from 4 spaces to 2? It seems to be the convention for `.html` files in Foundation site codebase.